doc/user/LIMITATIONS_AND_TROUBLESHOOTING.md

```diff
@@ -114,7 +114,9 @@ Your previous F3D installation is current being used to generate thumbnails, wai
 
 > When running F3D without interactivity (e.g. `f3d.exe --version`) from CLI, nothing is displayed.
 
-It's a limitation from the Windows "subsystem". You can workaround the issue by piping to a file (`f3d.exe --version > file.txt`) and read the file.
+It's a limitation from the Windows "subsystem".
+You can workaround the issue by piping to a file (`f3d.exe --version > file.txt`) and read the file.
```

Comment:
```suggestion
You can workaround the issue by redirecting output to a file and printing the file:
`f3d.exe --version > file.txt` && cat file.txt`
```

Please make sure `cat` works in cmd though :)

Reply:
it doesn't

Reply:
well it would be nice to find a command that works because we will need to update the issue template accordingly. I will take care of it